yarn.lock

```diff
@@ -3935,7 +3935,7 @@ debug@^4.1.0, debug@^4.1.1:
   dependencies:
     ms "2.1.2"
 
-debuglog@*, debuglog@^1.0.1:
```

Comment:
I have nfi why this lock file has changed - guessing it happened as part of running the build script. Lmk if I should revert this particular change.